### ISSUE TITLE
Scroll arrows positions should be reversed when direction=rtl

### ISSUE THREAD

**Issue by MWalid** [PolymerElements/paper-tabs#39]
With `direction=rtl` paper-tabs shows left arrow on right side (element start) while it's supposed to show it on the left side (element end), same for the right arrow. 

[Demo](http://codepen.io/MWalid/pen/KpZpMz)

**Comment by cdata** [comment]
Thanks for reporting! This is a pretty tricky problem.
